src/data_format/v0.rs

```diff
@@ -369,24 +374,19 @@ impl From<NetworkFilterV0DeserializeFmt> for NetworkFilter {
 
 #[derive(Debug, Deserialize, Default)]
 pub(crate) struct NetworkFilterListV0DeserializeFmt {
-    pub filter_map: HashMap<crate::utils::Hash, Vec<NetworkFilterV0DeserializeFmt>>,
+    pub flatbuffer_memory: Vec<u8>,
+    pub filter_map: HashMap<crate::utils::Hash, Vec<u32>>,
+    pub unique_domains_hashes_map: HashMap<crate::utils::Hash, u16>,
 }
 
 impl From<NetworkFilterListV0DeserializeFmt> for NetworkFilterList {
     fn from(v: NetworkFilterListV0DeserializeFmt) -> Self {
+        let _ = fb::root_as_network_filter_list(&v.flatbuffer_memory)
+            .expect("Flatbuffer is not corrupted");
```

Comment:
Is it result in `panic()` if the format doesn't match?
We definitely don't want to crash the browser process here.

Reply:
Implemented TryFrom instead